spec/vcr_cassettes/manageiq/providers/redhat/infra_manager/refresher_3_1.yml

```diff
@@ -2308,7 +2308,6 @@ http_interactions:
         rel=\"statistics\"/>\n        <host href=\"/api/hosts/2f1d11cc-e269-11e2-839c-005056a217db\"
         id=\"2f1d11cc-e269-11e2-839c-005056a217db\"/>\n        <network href=\"/api/networks/5fdbd265-0ead-4038-ba68-39ef4deafcd6\"
         id=\"5fdbd265-0ead-4038-ba68-39ef4deafcd6\"/>\n        <mac address=\"00:24:e8:69:f0:b6\"/>\n
-        \       <ip address=\"\" netmask=\"\"/>\n        <boot_protocol>static</boot_protocol>\n
```

Comment:
Why is this line deleted?  The raw response from the provider didn't change, did it?